### ISSUE TITLE
Christies super zoom (json dzi)

### ISSUE THREAD

**Issue by Tricesimo** [lovasoa/dezoomify#475]
### Site name and desciption
https://www.christies-media.com/assets/19872-62316484-sz/view/iframe

The webpage has both their regular zoom, but now also a super-zoom, called "BigObjectBase"

If I try using dezoomify using the web address, I get the error that it cannot find a dezoomable object. 

I have installed the dezoomify extension, and while it does appear to find the file:
https://www.christies-media.com/api/files/97b25d1e-14f4-4e9b-9a42-8fdafd81e420/superzoom.dzi
it gives me the same error as before (selected Seadragon)

I also downloaded the dezoomify-rs application, and I get the same errors each time:

ERROR Dezoomer error: Unable to create the dezoomer: Tried all of the dezoomers, none succeeded. They returned the following errors:

 - iiif: Unable to create the dezoomer: Invalid IIIF info.json file: missing field `width` at line 1 column 152
 - deepzoom: Unable to create the dezoomer: Unable to parse the dzi file: Syntax: 1:1 Unexpected characters outside the root element: {
 - krpano: Unable to create the dezoomer: Unable to parse the krpano xml file: Syntax: 1:1 Unexpected characters outside the root element: {

I wonder if it's a similar solution to how the Barnes website is working.

Thanks again for your help!

**Comment by lovasoa** [comment]
It looks like they use a json-encoded dzi file (instead of the usual xml). I'll mark this as "enhencement" and keep this issue open until we add support for it. Support for such files has been added recently in dezoomify-rs, but it currently requires having an `Url` property in the json.

**Comment by lovasoa** [comment]
![superzoom_2 copy](https://user-images.githubusercontent.com/552629/94810116-e2169880-03f3-11eb-8930-09287b578fbc.jpg)

**Comment by Shiraishi1** [comment]
Hello Tricesimo,
I am an avid fan of Vincent Van Gogh, and the works of Cente Van Gogh you listed above are so clear and delightful. I want to know how can I see the catalog of works so that I can see more works? We hope to get your help. Thank you

**Comment by Tricesimo** [comment]
Hi Shiraishi,

The website above is the Christie's website, which has many paintings but only when they come to auction. The two biggest auction houses that would normally auction works as expensive as van Gogh's are:

Christie's: 
https://www.christies.com/

Sotheby's:
https://www.sothebys.com/en/

You can search through their archived past sales and see what sort of items have been sold. 

The most famous ones are probably in art galleries, so here is where I would suggest you try:

Google Arts & Culture has many of Vincent's paintings (as well as many other famous artists):
https://artsandculture.google.com/

The Van Gogh Museum:
https://www.vangoghmuseum.nl/en

Art Institute of Chicago:
https://www.artic.edu/

The National Gallery, London:
https://www.nationalgallery.org.uk/paintings/search-the-collection

Metropolitan Museum of Art, New York:
https://www.metmuseum.org/art/collection

Museum of Modern Art (MoMA), New York:
https://www.moma.org/

National Gallery of Art, Washington DC:
https://www.nga.gov/collection/collection-search.html

The Barnes Collection:
https://collection.barnesfoundation.org/

There's many other of Van Gogh's works all over the world in various galleries. I use Google to look for the major art galleries in major cities around the world and can often find many wonderful works. Many of them can be downloaded from the website, or you can use the dezoomify website, dezoomify extension, or dezoomify-rs to extract them. 

Happy hunting!

**Comment by Shiraishi1** [comment]
Hello Tricesimo,
Thank you very much for your reply! !
This is awesome, thank you very much! !

**Comment by Tricesimo** [comment]
That is correct, but it doesn't always give you the Superzoom image. Christie's has been adding their images with a maximum dimension of 3200 pixels - any image that has either a height or width of 3200 with the other side shorter is a 'standard' image. 

When you can download a Superzoom, the image can have dimensions of 10,000+ but... often it's just the standard 3200 pixel image on a larger white background. Your image is an example of this. The standard image is 3200 x 2594, while the Superzoom is 4096 x 4096. But, the Superzoom is simply the standard image with a white border that extends to 4096 pixels. 

Here's an example of a painting that has both a standard 3200 pixel image, plus a true Superzoom image:
https://www.christies.com/lot/lot-6342708

In this case, the Superzoom image is 16,384 x 16,384, with the actual image being about 12,310 x 15,359. This image can be retrieved by launching the dezoomify app, copying the jpg file location that appears, and pasting it to dezoomify-rs.

Happy hunting!

**Comment by AlexanderDeLargeBrotherSir** [comment]
<img width="2048" height="2048" alt="Image" src="https://github.com/user-attachments/assets/b62853f1-28b4-4859-a526-266be0a46ccb" />

https://www.christies-media.com/api/files/97b25d1e-14f4-4e9b-9a42-8fdafd81e420/superzoom.dzi - This works now

@Tricesimo, please very kindly close this.
